tests/eden/network/testdata/test_networking.txt

```diff
@@ -4,7 +4,7 @@
 [!exec:curl] stop
 [!exec:sleep] stop
 
-# This test deploys two apps into EVE and checks access between them through network
+# This test deploys two apps into EVE and checks access between them trough network
```

Comment:
You introduced a typo here. Is that because the typo exists in eden repo and it doesn't run codespell?